internal/schemautil/custom_diff.go

```diff
@@ -116,6 +116,16 @@ func CustomizeDiffCheckPlanAndStaticIpsCannotBeModifiedTogether(_ context.Contex
 	return nil
 }
 
+// CustomizeDiffCheckStaticIpsExistence prevents users from modifying `aiven_static_ip` resources in
+// the same step with the resources that uses them due to Terraform limitation
+// See: https://terraform.io/plugin/sdkv2/resources/customizing-differences
+func CustomizeDiffCheckStaticIpsExistence(_ context.Context, d *schema.ResourceDiff, _ interface{}) error {
```

Comment:
Can you please add a test that validates this diff function? 
In one step, we should get this error `please modify 'aiven_static_ip' resource in a separate step`, and the next test step should fix that and showcase that it is gone.